JapaneseFilter/sections/specific.txt

```diff
@@ -35,6 +35,9 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+5chan.jp##div[class^="advertise_"] ([NSFW] https://c.5chan.jp/eu5FRL9Yr8)
```

Comment:
`([NSFW] https://c.5chan.jp/eu5FRL9Yr8)` must be removed.

Reply:
I see.

Reply:
Not yet resolved.

Reply:
yes. I will add.